internal/gonut/cmd/push.go

```diff
@@ -42,23 +42,27 @@ var GonutAppPrefix = "gonut"
 type sampleApp struct {
 	caption       string
 	buildpack     string
+	stack         string
 	command       string
 	aliases       []string
 	appNamePrefix string
 	assetFunc     func() (files.Directory, error)
 }
 
 var (
-	deleteSetting  string
-	summarySetting string
-	noPingSetting  bool
+	deleteSetting    string
+	outputSetting    string
+	buildpackSetting string
+	stackSetting     string
+	noPingSetting    bool
 )
 
 var sampleApps = []sampleApp{
 	{
 		caption:       "Golang",
 		command:       "golang",
 		buildpack:     "go_buildpack",
+		stack:         "cflinuxfs3",
```

Comment:
I would like to omit the stack here, because we have to maintain it if we specify it. If we leave it empty and use the `cf push ...` accordingly, we would pick the systems default stack if nothing is configured.